### PR TITLE
feat: profile-page, comment-box, emoji picker

### PR DESCRIPTION
**Pull request by VLeads**
This PR adds

- Profile Page layout
- Comment Box
- Comments
- Emoji picker
- Minor fixes

### CODE REVIEW

File: src/components/post/Post.jsx [hunk 1]
```diff
@@ -1,39 +1,110 @@
...
+              avatar={placeholder}
+              displayName={"user"}
+              username={"user1233"}
+              msg={"Hey Vishal !"}
+            />
+            <Comment
+              avatar={placeholder}
+              displayName={"Elon Musk"}
+              username={"elon1233"}
+              msg={" Vishal you are amazing !"}
```
Comment (partha8) [inline src/components/post/Post.jsx:103]: Nice! :joy:

**Review by partha8 — APPROVED** [review]
Everything looks good. Check the z-index of emoji container that you have as I was able to see "12" of the comment icon.

File: src/components/post/Post.jsx (hunk 1 above)
Comment (VLeads) [reply src/components/post/Post.jsx:103]: Thankyou 😁

**Review by anik31 — APPROVED** [review]
Good job, I liked the emoji picker you have added. Everything looks good & code is also clean.

**Comment by VLeads** [comment]
Thanks for the review.

**Pull request merged**